File: CredTodxs.Domain/Solicitacao.cs
using CredTodxs.Domain.Enums;

namespace CredTodxs.Domain
{
    public class Solicitacao
    {
        public int Id {get; set;}
        public int IdPessoa {get; set;}
        public Pessoa Pessoa {get; set;}
        public double QtdSolicitada {get; set;}
        public double RendaMensal {get; set;}
        public Residencia Residencia {get; set;}
        public bool Brasileiro {get; set;}
        public string CidadeNatal {get; set;}
        public string UfNatal {get; set;}
        public string NomeMae {get; set;}
        
    }
}

File: CredTodxs-APP/src/app/_models/Solicitacao.ts
import { Pessoa } from "./Pessoa";

export interface Solicitacao {

    id: number;
    idPessoa: number;
    pessoa: Pessoa;
    qtdSolicitada: number;
    rendaMensal: number;
    residencia: number;
    brasileiro: boolean;
    cidadeNatal: string;
    ufNatal: string;
    nomeMae: string;
}
